bin/cli.js

```diff
@@ -147,6 +154,35 @@ Example
       runTransform(target, command, program, { component, prop })
     );
 
+  program
+    .command(
+      "replace-prop-value <target> <component> <prop> <oldValue> <newValue>"
+    )
+    .description(
+      `
+Codemod used for prop value changing
+Usage
+  npx carbon-codemod replace-prop-value <target> <component> <prop> <oldValue> <newValue>
+
+  target       Files or directory to transform
+  component    Import path of component which prop should be renamed
+  attribute    Prop name
```

Comment:
Why was this renamed from `prop` to `attribute`? L165 still cals it prop.

Reply:
Because I'd called it `prop` here and `attribute` in `replace-prop-value.js` which meant it wasn't getting passed through initially so I had to change it in one place or the other. In `replace-prop-value.js` it's being passed into the `replaceAttributeValue` function so I decided just to call it attribute

I'll update that on line 165!